test/gmtregress/draper.sh

```diff
@@ -32,7 +32,7 @@ cat << EOF > draper.txt
 28.6	11.08
 EOF
 # First plot data and basic LS fit with equation
-txt=`gmt regress -Ey -N2 -Fxm -T25/80/2+n draper.txt -T0 | awk '{printf "85 13 @!y\\\\303 = %.4f %.4f x\n", $17, $15}'`
+txt=$(gmt regress -Ey -N2 -Fxm -T25/80/2+n draper.txt -T0 | awk '{printf "85 13 @!y\\\\303 = %.4f %.4f x\n", $17, $15}')
```

Comment:
```suggestion
txt=$(gmt regress -Ey -N2 -Fxm -T25/80/2+n draper.txt -T0 | awk '{printf "85 13 @!y\\303 = %.4f %.4f x\n", $17, $15}')
```
This will fix the draper.sh failure.